pkg/apis/playlist/v0alpha1/conversions.go

```diff
@@ -39,10 +39,6 @@ func convertToK8sResource(v *playlist.PlaylistDTO, namespacer namespaceMapper) *
 		})
 	}
 	return &Playlist{
-		TypeMeta: metav1.TypeMeta{
```

Comment:
The k8s infrastructure replaces whatever you put in here anyway, so it seems better to avoid it